tests/integration/targets/filesystem/tasks/main.yml

```diff
@@ -46,6 +46,10 @@
 
     # Other limitations and corner cases
 
+    # bcachefs only on Alpine > 3.18 and Arch Linux for now
+    # other distributions have too old versions of bcachefs-tools and/or util-linux (blkid for UUID tests)
+    - 'ansible_distribution == "Alpine" and ansible_distribution_version is version("3.18", ">") and item.0.key == "bcachefs"'
+    - 'ansible_distribution == "Archlinux and item.0.key == "bcachefs"'
```

Comment:
```suggestion
    - 'ansible_distribution == "Archlinux" and item.0.key == "bcachefs"'
```

Reply:
Sorry about that, I should have just gone to bed and look at it today.

Reply:
No worries :)